nav2_controller/src/nav2_controller.cpp

```diff
@@ -31,10 +31,14 @@ ControllerServer::ControllerServer()
 : LifecycleNode("controller_server", "", true),
   lp_loader_("nav2_core", "nav2_core::LocalPlanner")
 {
-  RCLCPP_INFO(get_logger(), "Creating controller");
+  RCLCPP_INFO(get_logger(), "Creating controller server");
 
   declare_parameter("controller_frequency", 20.0);
-  declare_parameter("local_controller_plugin", "dwb_core::DWBLocalPlanner");
+  std::vector<std::string> default_name, default_type;
+  default_type.push_back("dwb_core::DWBLocalPlanner");
+  default_name.push_back("FollowPath");
+  controller_names_ = declare_parameter("controller_plugin_names", default_name);
```

Comment:
If you agree with my suggestion above, then these names are not FollowPath, etc, but the actual controller names e.g. `DWBLocalPlanner, `MyAwesomeDocker`.